modules/rostests/apitests/crt/ceil.c

```diff
@@ -0,0 +1,109 @@
+/*
+* PROJECT:     ReactOS api tests
+* LICENSE:     MIT (https://spdx.org/licenses/MIT)
+* PURPOSE:     Tests for ceil / ceilf
+* COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+*/
+
+#define _CRTBLD // we don't want inline ceilf!
+#include "math_helpers.h"
+
+#ifdef _MSC_VER
+#pragma function(ceil)
+// ceilf is not available as an intrinsuc
```

Comment:
```suggestion
// ceilf is not available as an intrinsic
```
but yeah it "suc(c)s" :rofl:

Reply:
Fixed